app/services/widgets/widgets-data.ts

```diff
@@ -682,4 +668,13 @@ export const WidgetDisplayData = (platform?: string): { [x: number]: IWidgetDisp
     platforms: new Set(['twitch']),
     icon: 'icon-community',
   },
+  [WidgetType.GameWidget]: {
+    name: $t('Game Widget'),
+    description: $t('Let your viewers play a game in chat'),
+    demoVideo: false,
+    demoFilename: 'game-widget.png',
+    supportList: [],
+    platforms: new Set(['twitch']),
+    icon: 'icon-face-masks',
+  },
```

Comment:
I think we should eventually migrate all widget configuration to the single file. I created `widget-config.ts` for that but it's not connected in the SourcesShowcase yet

Reply:
i think there is a decent amount of separation of function between `WidgetsConfig` and `WidgetsDisplayData`. WidgetsDisplayData is only used in the SourceSelector and SourceShowcase, and cares about different things (like primary platform) which is unnecessary for WidgetsConfig, which appears to mostly handle how the widget actually functions and cares about things like the host and user token. it's technically two sources of truth, but if you look at it closely, there is zero overlap in function between them.

Reply:
Just look at the way how we add new widgets. The more files you should modify for a new widget the more complexity it brings. There is no action require for this PR here. But I'm going to switch widget to a single config in the future.